draft-ietf-quic-transport.md

```diff
@@ -1539,10 +1539,10 @@ magnitude of any amplification attack that can be mounted using spoofed source
 addresses.  In determining this limit, servers only count the size of
 successfully processed packets.
 
-Clients MUST pad UDP datagrams that contain only Initial packets to 1200 bytes.
-Once a client has received an acknowledgment for a Handshake packet it MAY send
-smaller datagrams.  Sending padded datagrams ensures that the server is not
-overly constrained by the amplification restriction.
+Clients MUST pad UDP datagrams that contain only Initial packets to at least
```

Comment:
Technically, we're padding the QUIC packet, not the UDP datagram.

Reply:
You could pad the datagram by sending a coalesced packet with only padding frames.